src/breadcrumb-group/styles.scss

```diff
@@ -64,3 +70,58 @@
 .breadcrumbs-skeleton {
   display: none;
 }
+
+.collapsed-button-dropdown {
+  display: block;
+  max-inline-size: 100%;
+}
+
+.collapsed-button {
+  @include styles.styles-reset;
+  @include styles.text-wrapping;
+  @include styles.font-button;
+  letter-spacing: awsui.$font-button-letter-spacing;
+  color: awsui.$color-text-interactive-default;
+  cursor: pointer;
+  padding-block: 0;
+  padding-inline: 0;
+  border-inline: none;
+  border-block: none;
+  background: none;
+  display: flex;
+  gap: awsui.$space-xxs;
+  max-inline-size: 100%;
+  @include focus-visible.when-visible {
+    @include styles.focus-highlight(awsui.$space-button-focus-outline-gutter);
+  }
+  &:hover {
+    color: awsui.$color-text-interactive-hover;
+  }
+  > :last-child {
+    color: awsui.$color-text-breadcrumb-current;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    &:hover {
+      color: awsui.$color-text-interactive-hover;
+    }
+  }
+}
+
+.button-icon-open {
```

Comment:
idea: Can we create some mixins to reuse the rotating icon styles between button dropdown, expandable sections, expandable rows, and this new component?

Reply:
Updated, at least for those that have a 180 spin. For those with 90degrees I think it's better to keep them separate from this as there's not so much direct overlap and they're a bit more complex with RTL.